template/contracts/.env.example

```diff
@@ -3,3 +3,4 @@ RPC_URL="https://sepolia.base.org"
 NETWORK="base_sepolia"
 SIGNATURE_MINT_SIGNER=
 BLOCK_EXPLORER_API_KEY="GET_API_KEY_FROM_THE_EXPLORER"
+BLOCK_EXPLORER="https://sepolia.basescan.org/"
```

Comment:
I believe you dont need to specify the BLOCK_EXPLORER, foundry is able to figure it out with your API KEY

Reply:
Since`base_sepolia` is an unknown alias for Foundry, when you're going to run any script, it will require to add at least one of `url` or `chain` to the Etherscan config. Again, check the Foundry book documentation: https://book.getfoundry.sh/reference/config/etherscan#etherscan

Reply:
This it the error you're going to get: `"At least one of 'url' or 'chain' must be present for Etherscan config with unknown alias 'base_sepolia'"`

Reply:
Thank you for sharing this @gabrielstoica.
Just out of curiosity, btw when in particular this error is happening?

Reply:
Don't worry @Zizzamia!

Considering the following configuration: 
```
[rpc_endpoints]
base_sepolia="${RPC_URL}"

[etherscan]
base_sepolia={key="${BLOCK_EXPLORER_API_KEY}"}
```

You'll get that error when running any script such as: `forge script script/BuyMeACoffee.s.sol --rpc-url base_sepolia`

Reply:
@gabrielstoica  Oh I see, I know why you are getting this error. You need to update your foundry version, they have made base_sepolia an officia alias now. 

You will not get any of the errors you have been happening and everything will work :)

Reply:
`base_sepolia` was recently introduced and a reason foundry's older version doesnt suppport it